schema/workbook.json

```diff
@@ -1,2 +1,754 @@
+                            "items": [
+                                {
+                                    "$ref": "#/definitions/metricItem"
+                                }
+                            ]
+                        },
+                        "filters": {
+                            "type": "array",
+                            "description": "Global filters applied to all selected metrics",
+                            "items": [
+                                {
+                                    "$ref": "#/definitions/metricFilters"
+                                }
+                            ]
+                        },
+                        "gridFormatType": {
+                            "type": "integer",
+                            "description": "The format in which the metric data should be displayed in the grid"
+                        },
+                        "resourceLimit": {
+                            "type": "integer",
+                            "description": "The maximum number of resources that can be queried"
+                        },
+                        "gridSettings": {
+                            "$ref": "#/definitions/gridSettings"
+                        },
+                        "exportToExcelOptions": {
+                            "$ref": "#/definitions/exportToExcelOptions"
+                        },
+                        "showExportToExcel": {
+                            "$ref": "#/definitions/showExportToExcel"
+                        },
+                        "title": {
+                            "$ref": "#/definitions/chartTitle"
+                        }
+                    }
+                },
+                "name": {
+                    "$ref": "#/definitions/name"
+                },
+                "customWidth": {
+                    "$ref": "#/definitions/customWidth"
+                },
+                "showPin": {
+                    "$ref": "#/definitions/showPin"
+                },
+                "styleSettings": {
+                    "$ref": "#/definitions/styleSettings"
+                },
+                "conditionalVisibility": {
+                    "$ref": "#/definitions/conditionalVisibility"
+                }
+            }
+        },
+        "link": {
+            "$id": "#/definitions/link",
+            "type": "object",
+            "required": [
+                "type",
+                "content"
+            ],
+            "properties": {
+                "type": {
+                    "const": 11
+                },
+                "content": {
+                    "type": "object",
+                    "properties": {
+                        "version": {
+                            "type": "string",
+                            "description": "Version of link step",
+                            "examples": [
+                                "LinkItem/2.0"
```

Comment:
some of these should just be default `version` not examples?  there's technically only one allowed version and its version is `LinkItem/1.0`